content/en/universal_service_monitoring/setup.md

```diff
@@ -0,0 +1,831 @@
+---
+title: Setting up Universal Service Monitoring
+kind: Documentation
+further_reading:
+- link: "/universal_service_monitoring/"
+  tag: "Documentation"
+  text: "Universal Service Monitoring"
+- link: "https://www.datadoghq.com/blog/universal-service-monitoring-datadog/"
+  tag: "Blog"
+  text: "Golden signals in seconds with Universal Service Monitoring"
+---
+
+
+## Supported versions and compatibility
+
+Required Agent version
+: Universal Service Monitoring requires that the Datadog Agent installed alongside your containerized service be at least version 6.40 or 7.40, higher as noted below for some beta features.
+
+Supported Linux platforms
+: Linux Kernel 4.14 and greater<br/>
+CentOS or RHEL 8.0 and greater
+
+Supported Windows platforms
+: IIS on Windows 2012 R2 and greater
+
+Supported application-layer protocols
+: HTTP<br/>
+HTTPS (OpenSSL)
+
+<div class="alert alert-info">
+Additional protocols and traffic encryption methods are supported in <a href="/universal_service_monitoring/additional_protocols/">private beta</a>. If you have feedback about what platforms and protocols you'd like to see supported, <a href="/help/">contact Support</a>.
+</div>
+
+## Prerequisites
+
+- If on Linux:
+    - Your service is running in a container.
+    - **Beta:** For non-containerized services see the [instructions here](#non-containerized-services-on-linux).
+- If on Windows and using IIS:
```

Comment:
```suggestion
- If on Windows IIS:
```
Based on the supported versions above, IIS is required. Edited cos the sentence might imply to some that it's optional.